device.mk

```diff
@@ -141,8 +134,30 @@ PRODUCT_CHARACTERISTICS := nosdcard
 
 DEVICE_PACKAGE_OVERLAYS := \
     device/lge/hammerhead/overlay
+    
+PRODUCT_PACKAGES += \
+    halium-boot \
+    libubuntu_application_api \
+    direct_ubuntu_application_sensors_c_api_for_hybris_test \
+    direct_ubuntu_application_sensors_for_hybris_test \
+    direct_ubuntu_application_gps_c_api_for_hybris_test \
+    libmedia_compat \
+    camera_service \
+    minimediaservice \
+    libaudioflingerglue \
+    libminisf \
+    minisfservice \
+    miniafservice
+    
+MINIMEDIA_SENSORSERVER_DISABLE := 1
 
+# for off charging mode
 PRODUCT_PACKAGES += \
+    charger_res_images
+
+PRODUCT_PACKAGES += \
+    brcm_patchram_plus \
```

Comment:
You can remove patchram and hciattach, the patched version is already in the root filesystem.

Reply:
Fixed